images/bazelbuild/Makefile

```diff
@@ -27,7 +29,10 @@ do_build = \
 do_push = \
 	gcloud docker -- push $(IMG):$(TAG)-$(VERSION);\
 	gcloud docker -- push $(IMG):latest-$(VERSION);\
-	echo Pushed $(IMG) with $(TAG)-$(VERSION) and latest-$(VERSION) tags;
+	echo Pushed $(IMG) with $(TAG)-$(VERSION) and latest-$(VERSION) tags;\
+	[[ $(VERSION) == $(LATEST_BAZEL_VERSION) ]];\
```

Comment:
maybe you need `ifeq`? I don't see a `latest-latest` tag :-)

Reply:
this works now, I wasn't tagging the image :confused: